Tasks/DotNetCoreCLIV2/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 2,
         "Minor": 145,
```

Comment:
please checkin generated changes in loc.json

Reply:
done